the-isme-journal.csl

```diff
@@ -5,6 +5,7 @@
     <id>http://www.zotero.org/styles/the-isme-journal</id>
     <link href="http://www.zotero.org/styles/the-isme-journal" rel="self"/>
     <link href="http://mts-isme.nature.com/cgi-bin/main.plex?form_type=display_auth_instructions" rel="documentation"/>
+    <link href="http://www.nature.com/ismej/ismej_new_gta.pdf" rel="documentation"/>
```

Comment:
The PDF link is where the actual documentation lives, but based on the filename I’m not confident that this may be not be renamed. The first link looks more stable/permalinky.